contrib/ci/fedora.sh

```diff
@@ -16,7 +16,16 @@ if [ "$QUBES" = "true" ]; then
     QUBES_MACRO=(--define "qubes_packages 1")
 fi
 
+# smoke test something small with no auto-deps
 meson setup .. \
+    -Dbuild=library \
+    -Dauto_features=disabled \
+    -Dtests=false $@
```

Comment:
do we actually need to pass arguments to this?

Reply:
Nope; removed, thanks.